sandbox-seed.json

```diff
@@ -3334,11 +3334,12 @@
     {
       "subject": "AMON Coincidence Alert from the sub-threshold IceCube-HAWC search NuEm-230927A",
       "submittedHow": "email-legacy",
-      "createdOn": 1695837395712,
+      "editedOn": 1695837395712,
```

Comment:
Both the `editedOn` and `createdOn` fields should be present in these examples.

Reply:
It is, this edit just corrects the key:
```json
    {
      "subject": "AMON Coincidence Alert from the sub-threshold IceCube-HAWC search NuEm-230927A",
      "submittedHow": "email-legacy",
      "editedOn": 1695837395712,
      "circularId": 34776,
      "submitter": "Hugo Ayala at Pennsylvania State University <hgayala@psu.edu>",
      "body": "...",
      "editedBy": "Example at Example <example@example.com>",
      "createdOn": 1694188746530,
      "version": 3
    },
```
The createdOn property should be consistent on all versions